### PR TITLE
Initial Commit

### PR DESCRIPTION
**Pull request by cs-harsh-sharma**
### Descriptions:
PMDB Link: [Click Here](https://pmdb.fortinet.com/ProjectManagement/editprojects.php?id=23659)

UTCs:
 - Connector installation verified.
 - Connector logo verified.
 - Actions and Playbook list verified.
 - Playbook tags verified.
 - All Playbook are in info mode verified.
 - All Playbook are in inactive mode verified.

Verified all actions.

### CODE REVIEW

File: manage-engine-key-manager-plus/info.json [hunk 1]
```diff
@@ -0,0 +1,222 @@
+{
+  "name": "manage-engine-key-manager-plus",
+  "label": "ManageEngine Key Manager Plus",
+  "description": "Manage Engine Key Manager Plus connector provides key management solution that helps you consolidate, control, manage, monitor, and audit the entire life cycle of SSH (Secure Shell) keys and SSL (Secure Sockets Layer) certificates.",
```
Comment (cs-utkarsh-surana) [inline manage-engine-key-manager-plus/info.json]: ManageEngine is one word

Comment (cs-utkarsh-surana) [reply manage-engine-key-manager-plus/info.json]: Do the same in all descriptions.

File: manage-engine-key-manager-plus/info.json [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+          "visible": true,
+          "type": "select",
+          "options": [
+            "Hostname/IP Address",
+            "IP Address Range"
+          ],
+          "name": "search_type",
+          "value": "Hostname/IP Address",
+          "tooltip": "Select one of the method to get SSL certificates across your network and import them into Key Manager Plus repository.",
+          "description": "Select one of the method to get SSL certificates across your network and import them into Key Manager Plus repository. You can choose between Hostname/IP Address or IP Address Range.",
```
Comment (cs-utkarsh-surana) [inline manage-engine-key-manager-plus/info.json]: As we are providing a default value of Hostname/IP Address can we same mention this in description.

File: manage-engine-key-manager-plus/info.json (hunk 1 above)
Comment (cs-harsh-sharma) [reply manage-engine-key-manager-plus/info.json]: done

File: manage-engine-key-manager-plus/info.json (hunk 2 above)
Comment (cs-harsh-sharma) [reply manage-engine-key-manager-plus/info.json]: done

**Review by cs-prerna-joshi — APPROVED** [review]

**Pull request merged**